src/web/backend/actions/api.py

```diff
@@ -0,0 +1,39 @@
+from flask import Blueprint
+from flask_restful import Api, Resource, reqparse
```

Comment:
We need this updated in Python requirements.

Reply:
forgot to commit it, added